prime-dotnet-webapi/Models/Api/GpidLookupOption.cs

```diff
@@ -0,0 +1,13 @@
+namespace Prime.Models.Api
+{
+    public class GpidLookupOption
+    {
+        /// <summary>
+        /// Used to pass parameters to api/provisioner-access/gpid-lookup
+        /// </summary>
```

Comment:
This is good but can it be moved to the class-level?

Reply:
comment moved to class level.